content/community/resources/cross-platform-development-with-python-and-beeware/contents.lr

```diff
@@ -0,0 +1,17 @@
+title: Cross-platform development with Python and BeeWare
+---
+author: Katie McLaughlin
+---
+description:
+
+Katie wrote an article about BeeWare and her PyCon US 2017 talk for OpenSource.com.
+
+`Cross-platform development with Python and BeeWare`_
+
+.. _Cross-platform development with Python and BeeWare: https://opensource.com/article/17/4/cross-platform-development-python-and-beeware`
```

Comment:
@glasnt could you please change this to 

https://opensource.com/article/17/4/cross-platform-development-python-and-beeware

(note the missing ` ' ` at the end)